src/main/java/uk/gov/hmcts/ccd/domain/model/search/SearchResultViewItem.java

```diff
@@ -16,9 +19,13 @@ public SearchResultViewItem() {
     }
 
     public SearchResultViewItem(final String caseId,
-                                final Map<String, JsonNode> caseFields) {
+                                final Map<String, JsonNode> caseFields,
+                                final List<CaseField> labelFields) {
         this.caseId = caseId;
         this.caseFields = caseFields;
+        labelFields
+            .stream()
+            .forEach(f -> caseFields.put(f.getId(), JsonNodeFactory.instance.textNode(f.getLabel())));
```

Comment:
The tests asserting the logic above are missing. Coverage is earned from other tests that are not directly related.

Reply:
Added SearchResultViewItemTest

Reply:
👍

Reply:
I think this logic belongs to MergeDataToSearchResultOperation. 
we should try to keep View type objects as simple as possible, ideally just plain data holders